snapcraft/internal/errors.py

```diff
@@ -119,6 +119,13 @@ def __init__(self, message):
         super().__init__(message=message)
 
 
+class SnapcraftDataDirectoryMissingError(SnapcraftReportableError):
```

Comment:
:+1:

Reply:
I added a new commit to also raise this in setup_dirs(), just in case no other path is taken...